### PR TITLE
Add display dashboard annotations

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolves: #186

### CODE REVIEW

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

File: src/hooks/useAnnotationEvents.ts [hunk 1]
```diff
@@ -43,22 +43,79 @@ const useAnnotations = (timeRange: TimeRange, options: CalendarOptions) => {
...
+          acc[curr.name] = curr.values;
+          return acc;
+        }, {} as DashboardAnnotation);
+
+        /**
+         * Return annotations
+         */
+        return Array.from(Array(annotation.length)).map((event, index) => {
+          return {
+            text: annotationObject.title[index],
```
[the remark this answers is not in the document]
Comment (mikhail-vl) [reply src/hooks/useAnnotationEvents.ts:73]:
```
react-dom.production.min.js:189 TypeError: Cannot read properties of undefined (reading '0')
    at eval (useAnnotationEvents.ts:73:41)
    at Array.map (<anonymous>)
    at eval (useAnnotationEvents.ts:71:53)
    at Array.flatMap (<anonymous>)
    at eval (useAnnotationEvents.ts:59:48)
    at gd (react-dom.production.min.js:244:332)
    at qd (react-dom.production.min.js:286:111)
    at react-dom.production.min.js:282:391
    at R (scheduler.production.min.js:13:203)
    at MessagePort.de (scheduler.production.min.js:14:128)
```

[the remark this answers is not in the document]
Comment (mikhail-vl) [reply src/hooks/useAnnotationEvents.ts:73]: 1. Add annotation query.
2. It should be displayed on the panel.
3. Click on Dashboard Refresh button.
4. Error displayed.

**Review by mikhail-vl — CHANGES_REQUESTED** [review]
Please update and resubmit.

File: src/hooks/useAnnotationEvents.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (vitPinchuk) [reply src/hooks/useAnnotationEvents.ts]: updated

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Comment by mikhail-vl** [comment]
@vitPinchuk Please add `text` as description in addition to title.

**Review by mikhail-vl — CHANGES_REQUESTED** [review]
Please add text to display additional information.

**Review by mikhail-vl — APPROVED** [review]
Looks good!

**Pull request merged**